corio.py

```diff
@@ -89,13 +94,14 @@ def parse_args():
     parser.add_argument("-ti", "--test_input", type=str,
                         help="Directory path containing test data input yaml files or "
                              "input yaml file path.")
-    parser.add_argument("-ll", "--logging-level", type=int, default=20,
+    parser.add_argument("-ll", "--logging-level", type=str, default="INFO",
                         help="log level value as defined below: " +
-                             "CRITICAL=50 " +
-                             "ERROR=40 " +
-                             "WARNING=30 " +
-                             "INFO=20 " +
-                             "DEBUG=10")
+                             "CRITICAL " +
+                             "ERROR " +
+                             'WARNING ' +
+                             'INFO ' +
+                             'DEBUG ' +
+                             'VERBOSE')
```

Comment:
Even though we have provided so many inputs e.g. Critical/Error/Warning/Info/Debug, how much useful are those?
Also we need to convert those and verify if input number given is correct or not. IMHO, this is all too much.
We can simplify this, instead of taking logging level as input, which is standard for any tool, i.e. just take `--debug` or `-d` option and use debug logging when that option provided. else use normal info logging.